### PR TITLE
Rename the kernel INSMassArtificialCompressibility to INSCompressibilityPenalty

### LINKED ISSUE #7276: Rename the kernel INSMassArtificialCompressibility to INSCompressibilityPenalty
To avoid confusing with the artificial compressibility method, the penalty method is renamed as INSCompressibilityPenalty.

### PR DESCRIPTION
**Pull request by fdkong**
Closes #7276

### CODE REVIEW

**Comment by jwpeterson** [comment]
@fdkong you need to rebase on upstream/devel before making these changes.  I fixed a couple issues with include guards and operator spacing, so this branch is currently in conflict.

**Comment by jwpeterson** [comment]
@fdkong take another look at this branch, it contains changes to 17 files, many of which you did not intend to make.

**Comment by jwpeterson** [comment]
Fix include guards in the header file to match the new filename.

File: modules/navier_stokes/include/kernels/INSCompressibilityPenalty.h [hunk 1]
```diff
@@ -11,20 +11,20 @@
 #include "Kernel.h"
 
 // Forward Declarations
-class INSMassArtificialCompressibility;
+class INSCompressibilityPenalty;
```
Comment (dschwen) [inline modules/navier_stokes/include/kernels/INSCompressibilityPenalty.h]: The include guard still says `INSMASSARTIFICIALCOMPRESSIBILITY_H`

**Comment by jwpeterson** [comment]
:+1:

**Pull request merged**